include/SFML/Window/Keyboard.hpp

```diff
@@ -29,6 +29,7 @@
 // Headers
 ////////////////////////////////////////////////////////////
 #include <SFML/Window/Export.hpp>
+#include <SFML/System/String.hpp>
```

Comment:
It's possible to do a forward declaration here.

Reply:
I'll keep it in mind when updating the PR, thanks!

Reply:
Done https://github.com/SFML/SFML/pull/1235/files#diff-1f2630fc4e70da1899bb04e88afe5ff00e3f8f5374e2b00b87c8dd260fd9bff0R36